tests/test_fixity.py

```diff
@@ -634,3 +636,39 @@ def test_scanall_if_sort_argument_is_passed(
             params={"username": STORAGE_SERVICE_USER, "api_key": STORAGE_SERVICE_KEY},
         ),
     ]
+
+
+@mock.patch("requests.get")
+def test_main_handles_exception_if_environment_key_is_missing(
+    _get: mock.Mock, mock_check_fixity: List[mock.Mock]
+) -> None:
+    _get.side_effect = mock_check_fixity
+    aip_id = uuid.uuid4()
+    stream = io.StringIO()
+    response = fixity.main(["scan", str(aip_id)], stream=stream)
```

Comment:
Since you're not using `aip_id` or `stream` in the rest of the test how about doing:

```suggestion
    response = fixity.main(["scan", str(uuid.uuid4())])
```